app/javascript/src/components/Common/Sidebar/index.jsx

```diff
@@ -1,67 +1,48 @@
 import React, { useState } from "react";
 
+import { LeftArrow, RightArrow } from "neetoicons";
 import { Sidebar as NeetoUISidebar } from "neetoui/layouts";
-import { useHistory } from "react-router-dom";
 
-import authenticationApi from "apis/authentication";
-import {
-  PROFILE_PATH,
-  CHANGE_PASSWORD_PATH,
-  LOGIN_PATH,
-} from "components/routeConstants";
-import { useAuthDispatch } from "contexts/auth";
-import { useUserState } from "contexts/user";
-
-import { APP_NAME, SIDENAV_LINKS } from "./constants";
+import { APP_NAME, SIDENAV_LINKS, AVATAR_IMAGE_URL } from "./constants";
 
 const Sidebar = () => {
   const [isSidebarCollapsed, setIsSidebarCollapsed] = useState(true);
-  const history = useHistory();
-  const authDispatch = useAuthDispatch();
-  const { user } = useUserState();
-
-  const handleLogout = async () => {
-    try {
-      await authenticationApi.logout();
-      authDispatch({ type: "LOGOUT" });
-      window.location.href = LOGIN_PATH;
-    } catch (error) {
-      logger.error(error);
-    }
-  };
 
   const bottomLinks = [
     {
       label: "My Profile",
-      onClick: () => history.push(PROFILE_PATH, { resetTab: true }),
-    },
-    {
-      label: "Change Password",
-      onClick: () => history.push(CHANGE_PASSWORD_PATH, { resetTab: true }),
+      // onClick: () => history.push(PROFILE_PATH, { resetTab: true }),
```

Comment:
The code that we merge to `master` branch is the code that is:
- Ready for deployment to production
- Ready to be worked on top of by other members of the team

Here what's the point of commenting out a piece of code and merging it to master? If you are commenting out something then it means, it's no longer needed. Thus delete the piece of commented code itself rather than pushing the commented code.